sdk/trace/src/main/java/io/opentelemetry/sdk/trace/samplers/SamplingResult.java

```diff
@@ -13,11 +13,13 @@
 import io.opentelemetry.api.trace.TraceState;
 import io.opentelemetry.context.Context;
 import java.util.List;
+import jdk.nashorn.internal.ir.annotations.Immutable;
```

Comment:
Looks like an accidental import

Reply:
Will fix it... No idea why idea decided to import this

Reply:
I think I've configured IDEA to never import anything from nashorn to avoid this.

Reply:
Done